magda-gateway/src/index.ts

```diff
@@ -166,7 +168,7 @@ const argv = addJwtSecretFromEnvVar(
         .options("enableAuthEndpoint", {
             describe: "Whether enable the AuthEndpoint",
             type: "boolean",
-            default: false
+            default: true
```

Comment:
Why do this? It's a bit scary, I'd say the `/auth` endpoint should definitely be off by default (really we should be removing it all together)

Reply:
It was an error. Fixed now.